src/js/components/AbilitiesBottomPanel.js

```diff
@@ -1,37 +1,42 @@
 import React, { Component } from 'react';
-import { connect } from "react-redux"
-import { saveAbilities } from '../actions/nextStepCharAction'
+import { connect } from "react-redux";
+import { bindActionCreators } from 'redux';
+import * as actionCreators from '../actions/nextStepCharAction';
+import "../../styles/attributes.scss";
 
-require("../../styles/attributes.scss");
+const AbilitiesBottomPanel = ({abilities, actions}) => {
+    return (
+        <div>
+            <button onClick={() => reRollAbilities(abilities, actions)}>REROLL</button>
+        </div>
+    );
+};
 
-@connect((store) => {
-    return {
-        abilities: store.nextStep.abilities
-    };
-})
-export default class AbilitiesBottomPanel extends Component {
-    saveAbilities() {
-        return this.props.dispatch(saveAbilities(this.props.abilities));
-    }
-    
-    getRandomInt(min, max) {
-        return Math.floor(Math.random() * (max - min + 1) + min);
-    }
-    
-    reRollAbilities() {
-        for(var key in this.props.abilities) {
-            if(this.props.abilities.hasOwnProperty(key)) {
-                this.props.abilities[key] = this.getRandomInt(9, 19);
-            }
+const saveAbilitiesToStore = (abilities, actions) => {
```

Comment:
same, you can just use
`(abilities, actions) => (actions.saveAbilities(abilities)`